src/containers/Home/Home.test.js

```diff
@@ -16,6 +17,7 @@ const homeTestWrapper = () => {
     <Provider store={testStore}>
       <BrowserRouter>
         <Home />
+        <Ecotip />
```

Comment:
Don't forget to remove the Ecotip stuff from this test file since those will be in a separate test file

Reply:
Good catch -- I can take it out with the next Ecotip test PR